docs/how-to-guides/how-to-scale-up-if-with-parallel.md

```diff
@@ -1,39 +1,51 @@
 ---
-title: Scaling Influence Computation
+title: How-to scale up influence function with parallelization
+alias:
+    name: scale-up-if-with-parallel
 ---
 
-# Scaling Influence Computation
+Calculating influence functions on large datasets can be challenging due to
+memory constraints (e.g. input data does not into memory or large models).
+In these cases, we want to map our influence function model over
+collections of batches (or chunks) of data.
 
-The implementations of [InfluenceFunctionModel][pydvl.influence.base_influence_function_model.InfluenceFunctionModel]
-provide a convenient way to calculate influences for
-in memory tensors. 
+This guide shows you how to parallelize influence function
+computations using pyDVL to handle large batches of data efficiently.
 
-Nevertheless, there is a need for computing the influences on batches of data. This might
-happen, if your input data does not fit into memory (e.g. it is very high-dimensional) or for large models
-the derivative computations exceed your memory or any combinations of these.
-For this scenario, we want to map our influence function model over collections of
-batches (or chunks) of data.
+!!! tip "Other guides"
 
-## Sequential
-The simplest way is to use a double for-loop
-to iterate over the batches sequentially and collect them. pyDVL provides the simple convenience class
-[SequentialInfluenceCalculator][pydvl.influence.influence_calculator] to do this. The
-batch size should be chosen as large as possible, such that the corresponding batches fit
-into memory.
+    For speeding-up data valuation algorithms with parallelization
+    see [[speed-up-value-with-parallel]].
+
+
+### Sequential Computation
```

Comment:
I think the sequential computation should be explained in a separate document that is marked as prerequisite for this one.